scripts/runnable_recipe_ray_vllm_weight_sync.py

```diff
@@ -250,6 +250,12 @@ def setup(self, cfg):
             actor_options={"num_cpus": 10, "num_gpus": 0},
             maxsize=self.cfg.vllm.queue_maxsize,
         )
+
+        # avoid sampling with replacement
```

Comment:
We don't want to avoid it all the time though. Maybe `log.info`? It's perfectly fine and expected to sample with replacement